python/hsfs/feature_group.py

```diff
@@ -903,6 +906,57 @@ def get_validations(self, validation_time=None, commit_time=None):
             self, validation_time, commit_time
         )
 
+    def compute_statistics(self, wallclock_time: Optional[str] = None):
```

Comment:
I think also here this should be in `FeatureGroupBase`

Reply:
I do this because this method is only for time-travel enabled feature groups. I wrote longer explanation above